src/cljs/ataru/virkailija/views/banner.cljs

```diff
@@ -56,6 +58,22 @@
       (< 1 org-count) "Useita organisaatioita"
       :else (-> organizations (first) :name (get-label)))))
 
+(defn- organization-rights-select []
+  (let [rights (subscribe [:state-query [:editor :organizations :rights]])]
+    [:div.profile__organization-rights-selector
+     "Valitse käyttäjän oikeudet"
+     (doall
+            (for [[right label] [[:view-applications "Hakemusten katselu"]
+                                 [:edit-applications "Hakemusten muokkaus"]
+                                 [:form-edit "Lomakkeiden muokkaus"]]]
```

Comment:
Tämän vois eriyttää komponentin luovaan funktioon (niin pääsisi for-loopista). Toisaalta vähän makuasia.